.github/CODEOWNERS

```
@@ -0,0 +1 @@
+ru.txt @Cjkjvfnby @o01eg
```

Please use an absolute path to the file as you are referring to that specific translation file.

```suggestion
/default/stringtables/ru.txt @Cjkjvfnby @o01eg
```

---

I reference all of them.  If for some reason this file will appear in the other place, I want to be a reviewer too.

---

@adrianbroher is that reasonable?

---

It is, but I personal would prefer explicit pathes due to their explicit nature, however a title comment like `# Code owners for translation stringtables` before this block would be fine too.  `ru.txt` will probably be unique within the tree, even when moving.

J

---

@Cjkjvfnby Could you add a title comment for the section as proposed?